notebooks/image_models/solutions/diffusion_model.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>I am not understanding the following phrases</p><p> </p><ul><li> "the variances of their noise components"</li><li> "The variances are required since denoising a signal requires different operations at different levels of noise."</li><li> " the model needs to learn how to denoise at different noise rates "</li></ul><p> </p><p>It would be useful to define the terms and expand the explanation. For instance, what does it mean for the model to de-noise at different rates? Do you mean for each different random image you need a different rate? Or do you mean at different steps of the de-noising you need different rates? If so, how do you chose these rates and why do you need different rates? (I don't think I actually understood what you meant so my questions are probably meaningless)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Reworded some things and added more explanation.</p>